### ISSUE TITLE
about retrain the spec2vec model

### ISSUE THREAD

**Issue by LXXiaogege** [iomega/spec2vec#79]

**Comment by LXXiaogege** [comment]
When I was training SPEC2vec, the loss value was 20000 +, is this normal？

**Comment by LXXiaogege** [comment]
Is there any format requirement for training data? Can only ues Peaks data be trained without metadata？
